src/backup/mpc/ManageKeyshares.tsx

```diff
@@ -70,4 +83,34 @@ ManageKeyshares.navigationOptions = () => {
     headerTitle: i18n.t('manageKeyshare'),
   }
 }
+
+const useAccountManagement = () => {
+  const goToUserKeyshare = () => {
+    // @note has seen education?
+    navigate(Screens.KeyshareEducationScreen, { type: KeyshareType.User })
+    // @todo nav to user keyshare education
+  }
+
+  const goToRecoveryKeyshare = () => {
+    // @note has seen education?
+    navigate(Screens.KeyshareEducationScreen, { type: KeyshareType.Recovery })
+    // @todo nav to restore education
+  }
+
+  const goToMultiParty = () => {
+    navigate(Screens.MultiPartyEducationScreen)
+  }
+
+  const { t } = useTranslation()
+  return useMemo(() => {
+    return [
+      { image: manageMigrate, action: goToUserKeyshare },
+      { image: manageRefresh, action: goToRecoveryKeyshare },
+      { image: manageMpc, action: goToMultiParty },
+    ].map((item, i) => {
+      return { ...item, title: t(`manageAccount.${i}`) }
+    })
+  }, [])
```

Comment:
Consider that if the user has seen the various education spaces, this should affect the memoization of the navigation functions.